bridges.go

```diff
@@ -102,11 +101,22 @@ func (bridgeType BridgeType) NotificationTargets() []BridgeUpdateNotification {
 }
 
 func (bridgeType BridgeType) FormatImage(image, commit string) string {
-	template, ok := imageTemplateOverrides[bridgeType]
+	templateString, ok := imageTemplateOverrides[bridgeType]
 	if !ok {
-		template = DefaultImageTemplate
+		templateString = DefaultImageTemplate
 	}
-	return fmt.Sprintf(template, image, commit)
+
+	var result bytes.Buffer
+	tmpl := template.Must(template.New("t").Parse(templateString))
+	err := tmpl.Execute(&result, map[string]string{
+		"Image":  image,
+		"Commit": commit,
+	})
+
+	if err != nil {
+		log.Fatalf("Failed to format image for %q", bridgeType)
```

Comment:
Any reason for using `%q` instead of `%s` here?

Reply:
Just copying from above